### PR TITLE
Fifth milestone

### PR DESCRIPTION
**Pull request by orestis-kaplan**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -2,19 +2,32 @@ import React from 'react';
...
+      Object.assign({}, prevState, calculate(prevState, buttonName))
+    );
   }
 
   render() {
     return (
       <div id="app" className="app-class">
-        <Display />
-        <ButtonPanel />
+        <Display result={this.state.total} />
```
Comment (SunnyAsar) [inline src/components/App.js:26]: destructure total

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hello @userman95  👋 
- already made some comments on the milestone 4 so i wont repeat them here.
### Wonderful work again.
but we have a bug here:
Screenshot 2019-09-30 at 12 59 19 PM


`5+5 = 10`
Thats fine, but see what happens next:
Screenshot 2019-09-30 at 12 59 08 PM


the next value is taken as part of the value with the result. unless it's an operation that is selected next, it should be a new calculation that begins.

**Please make the changes and request another review**
well done so far!!

File: src/components/App.js (hunk 1 above)
Comment (orestis-kaplan) [reply src/components/App.js:26]: Hey Sunny , how are you ? what you mean with destructure total? In my current code i dont use the other variables so i just play around with total .

Comment (SunnyAsar) [reply src/components/App.js:26]: Hi @userman95 , i'm very well thank you.
```
 const { total } = this.state;
```
line 29 can be
```
 <Display result={total} />
```

Comment (orestis-kaplan) [reply src/components/App.js:26]: oh alright ! thanks

**Review by BalintMendli — CHANGES_REQUESTED** [review]
Hi @userman95 ,

Great job fixing the issue from the previous review.

We still have a couple of issues we have to fix:

* Dividing by 0 blows up the app. Let's make sure we handle this.
* If I click on an operator and then another operator the app errors out. We need to take care of this also.
* I can enter any number of `.`s in a number which also causes a crash. Make sure we can only input one `.` in a number.
* Also the Display should show '0' by default. There's nothing shown now.

Please handle these issues and submit another review request.

If you have any questions or concerns do not hesitate to contact me on Slack.

**Pull request merged**